packages/admin-stories/src/admin/error-handling/ErrorDialogApolloProviderError.tsx

```diff
@@ -0,0 +1,100 @@
+import { ApolloClient, ApolloLink, ApolloProvider, createHttpLink, InMemoryCache, useQuery } from "@apollo/client";
```

Comment:
This story doesn't belong here.

Reply:
this is a force-push mixups I guess

Reply:
yes, that's because of some force pushes. i make a rebase